episodes/09-formative-assessment.md

```diff
@@ -42,6 +20,59 @@ exercises: 35
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+
+As we have seen previously, defining objectives for a lesson (or a teaching episode) can help to focus your content on the most important learning outcomes and outline the scope of your lesson project. With learning objectives you have defined your **intended (or planned) curriculum** - what you set out to accomplish. What skills and knowledge your learners actually leave with and can demonstrate after the workshop - the **attained curriculum** - will be a combination of **implemented** and **hidden** curricula:
+
+  - **implemented curriculum** are the concepts, relationships and skills that are explicitly covered in your lesson and translated into classroom practices - i.e. the teaching & learning activities,
```

Comment:
```suggestion
  - **implemented curriculum** is the concepts, relationships and skills that are explicitly covered in your lesson and translated into classroom practices - i.e. the teaching & learning activities,
```
curriculum/curricula terms are difficult to use the right matching singluar/plural verb...